package-lock.json

```diff
@@ -1,17 +1,17 @@
 {
     "name": "aws-vsts-tools",
-    "version": "1.10.0",
+    "version": "1.11.0",
```

Comment:
Looks like the version is still elevated in the package-lock.

Reply:
thanks for the help. it's been a while since I've worked with node. I ended up running this on npm version 7.15.0 and it added some extra attributes in package-lock